docs/phdata.rst

```diff
@@ -266,51 +268,51 @@ The **/setup** group contains information about the measurement setup:
-  receive the same spectral band **and** polarization state. For example,
+- **num_split_ch**: (integer) number of distinct detection channels
+  detecting the same spectral band **and** polarization state. For example,
   when a non-polarizing beam-splitter is employed in the detection path,
-  this value is 2. When no polarization- and spectral-insensitive splitting
-  is performed this value is 1.
+  its value is 2. When no splitting
+  is performed, its value is 1.
 
-- **modulated_excitation**: (boolean) *True* (i.e. 1) if there is any form of
-  excitation modulation either in wavelength (like in μs-ALEX or PAX) or in
-  polarization. This field is also *True* for pulse-interleaved excitation
+- **modulated_excitation**: (boolean) *True* (or 1) if there is any form of
+  excitation modulation either in the wavelength space (as in μs-ALEX or PAX) or in
+  the polarization space. This field is also *True* for pulse-interleaved excitation
   (PIE) or ns-ALEX measurements.
 
-- **lifetime**: (boolean) *True* (i.e. 1) if the measurements includes a
-  *nanotimes* array of (usually sub-ns resolution) photon arrival times
-  respect to a laser pulse (like in TCSPC measurements).
+- **lifetime**: (boolean) *True* (or 1) if the measurements includes a
+  *nanotimes* array of (usually sub-ns resolution) photon arrival times with
+  respect to a laser pulse (as in TCSPC measurements).
 
 - **excitation_wavelengths**: (array of floats) list of excitation wavelengths
-  (center wavelength if broad-band) in increasing order. Units are in *meters*.
+  (center wavelength if broad-band) in increasing order (unit: *meter*).
 
 - **excitation_cw**: (array of booleans) for each excitation source,
-  this field indicates whether it is continuous wave (CW), *True*, or pulsed,
+  this field indicates whether excitation is continuous wave (CW), *True*, or pulsed,
   *False*.
-  The order of excitation sources is the same as in
-  ``excitation_wavelengths`` and it is in increasing order of wavelengths.
+  The order of excitation sources is the same as that in
+  ``excitation_wavelengths`` and is in increasing order of wavelengths.
 
-The following fields are optional and not necessarly relevant for each
-particular experiment. If not-relevant these fields are omitted.
+The following fields are optional and not necessarly relevant for all experiments.
+If irrelevant, these fields are (**or can be?**) omitted.
 
 - **excitation_polarizations**: (arrays of floats) list of polarization
   angles (in degrees) for each excitation source.
   The order of excitation sources is the same as in
-  ``excitation_wavelengths`` and it is in increasing order of wavelengths.
+  ``excitation_wavelengths`` and is in increasing order of wavelengths.
 
 - **excitation_powers**: (array of floats) excitation power in *Watts*
-  for each excitation source.
+  for each excitation source (**this should really be an intensity and should be expressed in Watts/m^2**).
 
```

Comment:
What about when you don't know the intensity in the sample but you just measure the power level before entering the microscope? Still the input power can be valuable to compare different measurements.